web/src/beta/components/fields/SelectField/index.tsx

```diff
@@ -124,6 +126,8 @@ const PickerWrapper = styled(Popover.Content)`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  /* TODO: Fix this */
+  z-index: 700;
```

Comment:
This is required so the select field options can be shown on the UI.

Reply:
src/services/theme/reearthTheme/common/zIndex.ts

Although this file zindexes, the z-index for dropdown for dropdown is 600 which is not enough here.